.travis.yml

```diff
@@ -2,23 +2,17 @@ language: java
 jdk:
   - oraclejdk8
 services:
-  - mongodb
-addons:
-  postgresql: "9.3"
+  - docker
 env:
   matrix:
     - MODULE=!querydsl-scala PROFILES=all,travis,examples
     - MODULE=!querydsl-jpa-codegen PROFILES=jpa,hibernate5,travis,examples
     - MODULE=querydsl-scala PROFILES=all,travis
-install:
-  - mvn -B -q --projects ${MODULE} -P${PROFILES} install -DskipTests=true
+install: /bin/true
```

Comment:
This saves us several minutes on the build.

Reply:
How so? Or is that done in-docker?
As long as the build output (due to artifact download) isn't too crazy it's 👍

Reply:
It saves time because we don't have to run a full `mvn install` before starting the tests. We don't need those artifacts so using `/bin/true` effectively skips that step. If you omit `install:`, travis will add a `mvn install` for you.